api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters-blm/asset-factory.service.ts

```diff
@@ -17,15 +22,43 @@ export class AssetFactory {
 
     copySync(from.workingDirectory, to.workingDirectory, {
       // Marxan binary is already linked
-      filter: (src, dest) => !src.match(/marxan/),
+      filter: (src) => !src.match(/marxan/),
     });
 
-    // TODO modify input.dat file - replace "BLM" with overrideBlmValue
+    const inputDat = this.getInputDat(assets);
+    if (!inputDat) throw new Error('No URL for input.dat found');
+
+    const inputDatPath = resolve(
+      to.workingDirectory,
+      inputDat.relativeDestination,
+    );
+
+    await this.ensureWriteDirectoryExists(inputDatPath);
```

Comment:
is this necessary? `copySync` should already handle the structure 🤔

Reply:
Fixed, I've remove that line